package.json

```diff
@@ -14,7 +14,7 @@
     "type": "git",
     "url": "git://github.com/mapbox/geojson-vt.git"
   },
-  "main": "src/index.js",
+  "main": "geojson-vt-dev.js",
```

Comment:
Why? If you're using browserify or rollup or etc, it'll handle the `main` file just fine. I don't see any reason to include `browser` or change `main` of this project. Running `browserify` on prepublish will be nice if we want an `npmcdn` workflow, but it should be tightly scoped to that need, not change essential stuff like the `main` entry